src/components/SidePanels/TabPanels/utils.ts

```diff
@@ -231,6 +231,27 @@ export const partialEmptyBaseProps = {
   panelViews: [panelViews[0], panelViews[1], panelViews[2], panelViews[3], panelViews[4]],
 };
 
+export const openSourcePartialEmptyBaseProps = {
```

Comment:
Instead of duplicating this, just to not include the 1 panel view, we should just conditionally include it, or just filter it out.